File: app/src/main/java/com/example/plantea/presentacion/CrearPlanInterface.java
package com.example.plantea.presentacion;

public interface CrearPlanInterface {

    //CategoriasFragment
     void mostrarCategoria(int idCategoria);
     void mostrarsubCategoria(String tituloCategoria);


    //CategoriasPictogramasFragment
    void cerrarFragment();
    void nuevoPictogramaDialogo();
    void pictogramaSeleccionado(String titulo, String imagen, int categoria);


}

File: app/src/main/java/com/example/plantea/presentacion/CrearPlanInterface.kt
package com.example.plantea.presentacion

interface CrearPlanInterface {
    //CategoriasFragment
    fun mostrarCategoria(idCategoria: Int)
    fun mostrarsubCategoria(tituloCategoria: String?)

    //CategoriasPictogramasFragment
    fun cerrarFragment()
    fun nuevoPictogramaDialogo()
    fun pictogramaSeleccionado(titulo: String?, imagen: String?, categoria: Int)
}